### PR TITLE
It's a morgue, not a rec room.

### PR DESCRIPTION
**Pull request by ProperPants**
The bartender, chef, and botanist all have morgue access. I don't know why or when, but this access is functionally useless to the core of their jobs.

**Bartender**
I have removed morgue access from the bartender because I truly see no reason they should have it. Maybe it has a tiny impact to bartender changelings or something? Even then, it would put them in the same boat as the other 90+% of the station that can't just stroll into the morgue.

**Chef**
I have left the chef with morgue access because the morgue is a prime source of... exotic ingredients. It seems appropriate/funny, especially if you are trying to RP someone like Hannibal Lecter.

**Botanist**
I have removed morgue access from the botanist. The only conceivable reason that the botanist should be in the morgue is to get blood for replica pods; however, the chance of someone being cloned via replica pod is so low, that if the people who are actually in charge of cloning (doctors, geneticists, coroner) want it to happen, then can just give a syringe of blood to botany.

🆑 ProperPants
tweak: Removes morgue access from bartender.
tweak: Removes morgue access from botanist.
/ 🆑

### CODE REVIEW

File: code/game/jobs/job/support.dm [hunk 1]
```diff
@@ -9,7 +9,7 @@
 	supervisors = "the head of personnel"
 	department_head = list("Head of Personnel")
 	selection_color = "#dddddd"
-	access = list(access_hydroponics, access_bar, access_kitchen, access_morgue, access_weapons, access_mineral_storeroom)
+	access = list(access_hydroponics, access_bar, access_kitchen, access_weapons, access_mineral_storeroom)
```
Comment (tigercat2000) [inline code/game/jobs/job/support.dm]: This does nothing. We use minimal_access exclusively.

**Comment by Aurorablade** [comment]
Botany can clone coprses...should be kept. 
Chef can make meat out of corpses.

...Dunno whats up with bartender...but thats funny.

**Comment by Citinited** [comment]
Podmen is not currently in code

**Comment by taukausanake** [comment]
@Citinited I'm pretty sure it was readded with Fox's port of Botany. I could be wrong though

**Comment by Citinited** [comment]
@taukausanake It's unticked in my most recent Github .dme, ergo it's not in-game.

**Comment by Fox-McCloud** [comment]
`access_list` isn't currently used, so you're giving a false impression of what's being removed here.

The reason `access` is so broad, is it's meant to be used during low pop, so that one job can fill many roles (ie: there's 20 players on and you get a bartender, but you also need a chef; the bartender can fill that role).

Under that case it becomes quite clear; the chef has (or had) morgue access; so it makes sense the bartender would also have this. 


I do, however, take issue with removing morgue access from hydroponics; the reason it was given to them is because of pop people. While yes, the chances of being cloned in this manner are low, I don't think botanists should be excluded the opportunity to do so, especially now that the map bug of "morgue access = basic medical access" has been resolved.

If botanist is to have morgue access, then chef and bartender also nee morgue access under their `access` list for this reason; if `access` is ever utilized, it means it's likely low pop, where botanist/chef/bartender roles are interchangeable.

**Comment by Aurorablade** [comment]
its in the code
https://github.com/ParadiseSS13/Paradise/blob/b496763145d03047ea63c29708844891e6671fc9/code/modules/hydroponics/grown/replicapod.dm#L56
https://github.com/ParadiseSS13/Paradise/commit/c5f3cf94e9cdb285653510403c0876e16832c6bf

**Comment by Citinited** [comment]
Yes, but it's not checked in the .dme, so it doesn't get loaded at roundstart.

At least that's how I understand it.

**Comment by Aurorablade** [comment]
that file is dcrepicated and does not control the pod people clonding. The links i linked do. and the code is currently active, on the server, at round start.

**Comment by Fox-McCloud** [comment]
https://github.com/ParadiseSS13/Paradise/blob/master/code/modules/hydroponics/grown/replicapod.dm

Pod people are very much a thing and are very much compiled in.

**Comment by ProperPants** [comment]
@Fox-McCloud I did not know how the two difference accesses worked, so thank you for the information. I was assuming that minimum access was some kind of emergency stuff, while access_list was normal access.

I'm pretty neutral on botanists having morgue access. To be honest, the primary reason I made this PR was just to get comfortable with the mechanics of making commits and pushing them.

**Comment by Fox-McCloud** [comment]
>To be honest, the primary reason I made this PR was just to get comfortable with the mechanics of making commits and pushing them.

If this is your primary notification, Para's main git shouldn't be your practice grounds, nor should you be making PRs for practice that have actual ramifications; if you're truly just wanting to practice, then you can make a dummy branch on your fork and commit to that, makes changes, and make pull requests to and from that dummy fork to another dummy fork.

End run: your master branch is pure, you're not submitting non-useful PRs to Para, and you're getting practice.

**Comment by SamHPurp** [comment]
With a dedicated coroner role, I agree that these guys shouldn't have access to it. The coroner is in charge of the body and can discharge bodies into the Botanists possession.

They shouldn't be able to just waltz in and take them.

**Pull request closed without merging**